components/pages/people/relationships/view/Relationships.tsx

```diff
@@ -67,21 +68,66 @@ const Relationships = ({ id }: Props): React.ReactElement => {
 
         <hr className="govuk-divider" />
         {personalRelationships && personalRelationships.length > 0 ? (
-          <dl className="govuk-summary-list lbh-summary-list">
-            {personalRelationships
-              .sort((a, b) => b.type.localeCompare(a.type))
-              .map((relationship) => {
-                if (relationship.persons.length > 0) {
-                  return (
-                    <RelationshipsGroupedByType
-                      type={relationship.type}
-                      persons={relationship.persons}
-                      key={`rel_${relationship.type}`}
-                    />
-                  );
-                }
-              })}
-          </dl>
+          <table className="govuk-table lbh-table">
+            <thead className="govuk-table__head govuk-visually-hidden">
+              <tr className="govuk-table__row">
+                <th scope="col" className="govuk-table__header">
+                  Relationship type
+                </th>
+                <th scope="col" className="govuk-table__header">
+                  Name
+                </th>
+              </tr>
+            </thead>
+            <tbody className="govuk-table__body">
+              {personalRelationships
+                .sort((a, b) => b.type.localeCompare(a.type))
+                .map((relationship) => {
+                  return relationship.persons
+                    .sort(
+                      (a, b) =>
+                        a.lastName.localeCompare(b.lastName) ||
+                        a.firstName.localeCompare(b.firstName)
+                    )
+                    .map((person, personRowIndex) => (
+                      <tr
+                        className="govuk-table__row"
+                        key={`${relationship.type}-${personRowIndex}`}
+                      >
+                        {personRowIndex === 0 && (
+                          <th
+                            scope="row"
+                            className="govuk-table__header govuk-!-width-one-quarter"
+                            rowSpan={relationship.persons.length}
+                          >
+                            {getTitleString(
+                              relationship.type as keyof typeof relationshipTypeMappings
+                            )}
+                          </th>
+                        )}
+                        <td
+                          data-testid={`related-person-name-${personRowIndex}`}
+                          key={`related-person-name-${personRowIndex}`}
+                          className={`${
+                            relationship.persons.length > 1 &&
+                            personRowIndex !== relationship.persons.length - 1
+                              ? `govuk-table__cell govuk-!-width-three-quarters ${style.noBorder}`
+                              : 'govuk-table__cell govuk-!-width-three-quarters'
+                          }`}
```

Comment:
Any ideas on how to do this better and with pure CSS? 😅

Reply:
If you want to target the last element only, you can use the  ":last-child" css selector in the rule definition to change the style of the last row only - pretty sure this is fine tho